plugins/org.eclipse.reddeer.core/src/org/eclipse/reddeer/core/lookup/WidgetLookup.java

```diff
@@ -114,6 +115,17 @@ public static WidgetLookup getInstance() {
 		return (T)found.getResult();
 	}
 	
+	private String createRererencedCompositeDebugMsg(ReferencedComposite refComposite){
+		StringBuilder sb = new StringBuilder();
+
+		if (refComposite == null){
+			sb.append(" no referenced composite specified");
+		} else {
+			sb.append(" referenced composite '"+refComposite.getClass()+"'");
```

Comment:
Please change appended text to something like " with referenced composite". So it would result in "Looking up active widget with class type some.package.Widget with referenced composite some.other.package.CompositeClass....". Maybe also remove apostrophes, it would look better (class name in whole output is without apostrophes.